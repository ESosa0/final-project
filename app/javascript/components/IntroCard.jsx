import React from 'react'
import Line from '../../assets/images/line.svg'
import WhiteBottle from '../../assets/images/wine_bottle_white_question.svg'
import PinkBottle from '../../assets/images/wine_bottle_pink_question.svg'
import RedBottle from '../../assets/images/wine_bottle_red_question.svg'
import classNames from 'classnames'
import Card from './Card'


const IntroCard = ({onNextClick}) => {
  return (
    <Card
      text={"Pair My Dish"}
      buttonText={"Get Started"}
      onNextClick={onNextClick}
      shouldScroll={false}
      blurb={"We choose the perfect wine for your dish using pairing techniques from top sommeliers."}
    >
      <img src={WhiteBottle} className="wine-bottle"/>
      <img src={PinkBottle} className="wine-bottle"/>
      <img src={RedBottle} className="wine-bottle"/>
    </Card>
  )
}

export default IntroCard
