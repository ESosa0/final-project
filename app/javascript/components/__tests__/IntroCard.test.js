import IntroCard from '../IntroCard'
import {shallow} from 'enzyme'
import toJSON from 'enzyme-to-json'
import React from 'react'
import {connect} from 'react-redux'

describe("IntroCard Component", () => {
  it("renders a default intro card", () => {
    const component = shallow(<IntroCard/>)
    expect(toJSON(component)).toMatchSnapshot()
  })
})
