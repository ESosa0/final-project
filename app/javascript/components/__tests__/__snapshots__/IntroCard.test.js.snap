// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IntroCard Component renders a default intro card 1`] = `
<Card
  blurb="We choose the perfect wine for your dish using pairing techniques from top sommeliers."
  buttonText="Get Started"
  shouldScroll={false}
  text="Pair My Dish"
>
  <img
    className="wine-bottle"
    src="test-file-stub"
  />
  <img
    className="wine-bottle"
    src="test-file-stub"
  />
  <img
    className="wine-bottle"
    src="test-file-stub"
  />
</Card>
`;
